### PR TITLE
fix(amazonq): disabled prompt input

### PR DESCRIPTION
**Pull request by dogusata**
## Problem
[ISSUE](https://issues.amazon.com/issues/D106435645)
Screenshot 2023-11-30 at 21 18 54

When panel width is too short, wrapped placeholder causes the input feel like it doesn't get focus, however it got focus.
Placeholder should not wrapped to a new line, instead needs to use ellipsis.

## Solution
Updated mynah-ui version which contains the no-wrap ellipsis fix for prompt input's placeholder.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by dogusata** [comment]
> Yes, this needs a changelog. Please also check https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog for guidance on the changelog wording.

Added changelog item

File: .changes/next-release/Bug Fix-dc5d5825-4d0e-4a2e-9ec3-63b4a0466662.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "fixed input prompt gets wrapped to new line which avoids focusing"
```
Comment (justinmk3) [inline .changes/next-release/Bug Fix-dc5d5825-4d0e-4a2e-9ec3-63b4a0466662.json:3]: please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

Comment (justinmk3) [inline .changes/next-release/Bug Fix-dc5d5825-4d0e-4a2e-9ec3-63b4a0466662.json:3]:
```suggestion
	"description": "Input prompt gets wrapped to new line which avoids focusing"
```

File: .changes/next-release/Bug Fix-dc5d5825-4d0e-4a2e-9ec3-63b4a0466662.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Input prompt gets wrapped to new line which avoids focusing"
```
Comment (justinmk3) [inline .changes/next-release/Bug Fix-dc5d5825-4d0e-4a2e-9ec3-63b4a0466662.json:3]:
```suggestion
	"description": "Amazon Q: Input prompt gets wrapped to new line which avoids focusing"
```

**Pull request merged**